src/index.js

```diff
@@ -57,11 +57,16 @@ export default class ValidatorDomainVerifier {
     const valPubKey = new ValPublicKey(validationPublicKey)
     const account_id = valPubKey.getAddress()
     ValidatorDomainVerifier._validateRippleAddress(account_id)
```

Comment:
I need to either stop throwing `InvalidRippleAccount` in this check or add that error as a valid verification error here:
https://github.com/ripple/validator-registry-api/blob/feature/verifications/api/models/verification.js#L15-L19